kura/examples/org.eclipse.kura.example.eddystone.scanner/.gitignore

```diff
@@ -0,0 +1 @@
+/bin/
```

Comment:
I didn't noticed that file. IMHO it is not needed for this example and also for iBeacon scanner. I'll delete them.